app/controllers/candidate_interface/personal_details_controller.rb

```diff
@@ -1,5 +1,7 @@
 module CandidateInterface
   class PersonalDetailsController < CandidateInterfaceController
+    before_action :authenticate_candidate!
```

Comment:
Talked about IRL: we'll switch to authenticating by default, and skipping where appropriate.